volume-providers/vpc/util/unit_util.go

```diff
@@ -0,0 +1,225 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package util
+
+import (
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils/reasoncode"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// Constants
+const (
+	DiskUnitsMB string = "MB"
```

Comment:
Please check these const values are useful or not, I don't think so its useful for volume

Reply:
DONE